changelog.d/20240110_101228_kyle_importnewdemocourse.md

```diff
@@ -0,0 +1,2 @@
+- 💥 [Feature] `tutor local do importdemocourse` will now import the new [Open edX Demo Course](https://github.com/openedx/openedx-demo-course), which has been rebuilt from scratch. If you wish to import the old demo course instead (which no longer receives updates), you can run: `tutor local do importdemocourse --repo-dir . --version open-release/palm.4`.
```

Comment:
We should replace "palm.4" by "quince.1".

Reply:
Does this mean that you need to be on `quince.2` to get the new version of the course, or will it be possible to be on an older version of the platform but still import the new course?

Reply:
> We should replace "palm.4" by "quince.1".

Right, or even `quince.master`.

> Does this mean that you need to be on quince.2 to get the new version of the course, or will it be possible to be on an older version of the platform but still import the new course?

Older versions of Open edX and Tutor will *default* to the old demo course, but it is _possible_ to import the new demo course into them with the right arguments:

```
tutor local do importdemocourse --repo-dir demo-course/course --version master
```

I'll update this message to be clearer about this.